xl-v1-metadata.go

```diff
@@ -52,6 +52,12 @@ type checkSumInfo struct {
 	Hash      string `json:"hash"`
 }
 
+// Default bit-rot checksum technique used in version '1'.
+const (
+	bitRotAlgoV1 = "blake2b"
```

Comment:
> it will be incorrect to call it bitRotAlgoV1 because V1 format allows different objects to have different algos, i.e if we change the algo even then xl format is still V1

What it indicates is that we are using "blake2b" for V1 format.  Also while it is possible we are never going to write objects with different checksum algos with in a given instance of a running server - there is no facility in erasureCreateFile() to do that, we can read different algos though.